pkgs/tools/security/imhex/default.nix

```
@@ -0,0 +1,89 @@
+{ lib
+, stdenv
+, cmake
+, fetchFromGitHub
+, mbedtls
+, gtk3
+, pkgconfig
+, capstone
+, libGLU
+, glfw3
+, file
+, python3
+, curl
+, xlibs
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ImHex";
+  version = "1.11.1";
+
+  src = fetchFromGitHub {
+    fetchSubmodules = true;
+    owner = "WerWolv";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-3S6mQ/fmxT2m1oH/MRn+3egj+HPYzaZys82nZ3RcE1I=";
+  };
+
+  patterns_src = fetchFromGitHub {
+    owner = "WerWolv";
+    repo = "ImHex-Patterns";
+    rev = "47fc94928960397feba24dd687f9b1c803213b34";
+    sha256 = "sha256-bufoyXbqRS/hIOU03ACBfHdrATTnuW5R1IiNLF/B9t4=";
+  };
+
+  nativeBuildInputs = [ cmake python3 ];
+
+  buildInputs = [
+    mbedtls
+    gtk3
+    pkgconfig
+    capstone
+    libGLU
+    glfw3
+    file
+    curl.dev
+    xlibs.libXdmcp.dev
+  ];
+
+  configurePhase = ''
+    cmake -DCMAKE_BUILD_TYPE=Release .
+  '';
+
+  buildPhase = ''
+    make -j
+  '';
+
+  installPhase = ''
+    install -D imhex $out/bin/imhex
+    install -D plugins/builtin/builtin.hexplug $out/bin/plugins/builtin.hexplug
+    install -D plugins/libimhex/libimhex.so $out/lib/libimhex.so
+
+    cp -R ${patterns_src}/* $out/bin
+  '';
+
+  fixupPhase = ''
+    function mypatch {
+      MYRPATH=$(patchelf --print-rpath "$1" | sed 's/\/build\/source\/plugins\/libimhex:://g')
+      patchelf --set-rpath "$2:$MYRPATH" "$1"
+    }
+    mypatch $out/bin/plugins/builtin.hexplug  $out/lib
+    mypatch $out/bin/imhex $out/lib
+  '';
+
+  hardeningDisable = [ "format" ];
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Hexviewer";
+    longDescription = ''
+      A Hex Editor for Reverse Engineers, Programmers
+      and people who value their retinas when working at 3 AM.
+    '';
```

```suggestion
    description = "Hex Editor for Reverse Engineers, Programmers and people who value their retinas when working at 3 AM";
```
The program name is not a proper description. Also the long one is short enough to be the normal one.